exasol_integration_test_docker_environment/lib/base/ssh_access.py

```diff
@@ -10,20 +10,37 @@
 
 
 _LOCK_FILE = "$TMP/$MODULE-ssh-access.lock"
-_DEFAULT_FOLDER = "$TMP/$MODULE"
+_DEFAULT_FOLDER = "$HOME/.cache/exasol/$MODULE"
 
 
 def _path(template: str) -> Path:
-    module_name = "integration-test-docker-environment"
     return Path(
         Template(template)
         .substitute(
             TMP=tempfile.gettempdir(),
-            MODULE=module_name,
+            HOME=os.path.expanduser('~'),
+            MODULE="itde",
         )
     )
 
 
+class SshFiles:
+    def __init__(self, folder: Optional[Path] = None):
+        self._folder = folder if folder else _path(_DEFAULT_FOLDER)
+
+    @property
+    def folder(self) -> Path:
```

Comment:
I chose "folder" because it's shorter than "directory" and I like the pair of names file/folder both starting with "F" and of similar length. But other than that I am agnostic. 
Please tell me if you'd prefer me to rename all occurrences of "folder" to "directory".

Reply:
Probably, best we use a consistent wording, please check if we use directory or folder elsewhere in the project.

Reply:
I will prefer "directory".

Reply:
done